src/views/BaseView.vue

```diff
@@ -1,42 +1,51 @@
 <template>
-  <v-layout>
-    <!--          :clipped="$vuetify.breakpoint.lgAndUp"-->
+  <v-app>
     <v-navigation-drawer
-      color="sidebar"
-      :expand-on-hover="true"
+      v-model="drawer"
       permanent
       rail
-      role="navigation"
+      expand-on-hover
+      color="#378dc5"
```

Comment:
@mohamalnadi This color is defined in vuetify.ts, so you can use `color="tertiary"`. 
Also, I mistakenly added the `theme="dark"` here because the Vue2 version had the `dark` property. The navigation drawer is supposed to have a different color in light mode vs dark mode (at least that's how it works in QA running the Vue2 code), so the `dark` prop must not have had any effect. I think if you change the color to tertiary and remove the theme property, it should have the right background color in both light and dark.